scripts/index.js

```diff
@@ -0,0 +1,164 @@
+let currentTemp = 100;
+let isCelsius = false; 
+
+const updateTemperature = currentTemp => {
+    const tempContainer = document.getElementById("temp-integer"); 
+    tempContainer.textContent = currentTemp;
+    changeTempColor(currentTemp);
+    changeLandscape(currentTemp);
+};
+
+
+const increaseTemperature = () => { 
+    currentTemp += 1;
+    updateTemperature(currentTemp);
+};
+
+const decreaseTemperature = () => { 
+    currentTemp -= 1; 
+    updateTemperature(currentTemp);
+};
```

Comment:
Just a suggestion, you could combine these with:

```suggestion
const changeTemperature = (increment) => { 
    currentTemp += increment;
    updateTemperature(currentTemp);
};
```

Then in the click event you could

```javascript
tempIncreaseBtn.addEventListener("click", () => changeTemperature( +1 ) ); 

tempDecreaseBtn.addEventListener("click", () => changeTemperature( -1 ) ); 

```